packages/core/components/AutoField/index.tsx

```diff
@@ -72,7 +72,7 @@ export const FieldLabelInternal = ({
   children,
   icon,
   label,
-  el = "label",
+  el = "div",
```

Comment:
Changing the default is a breaking change. Is there an array-only fix for this?

Reply:
@chrisvxd for the docs bug yeah there is...but if anyone trying to do custom interface and used `<FieldLabel></FieldLabel>` with `<AutoField><AutoField/>` of type `Array` he must path the `el` prop as `"div"` otherwise he will face the same issue in his app thats why i changed the default value

Reply:
@chrisvxd for the docs bug yeah there is...but if anyone trying to do custom interface and used `<FieldLabel></FieldLabel>` with `<AutoField><AutoField/>` of type `Array` he must pass the `el` prop as `"div"` otherwise he will face the same issue in his app thats why i changed the default value

Reply:
I think we should fix the docs example, but not change the default behaviour.

`label` is the correct default, because it introduces correct form semantics.

The correct solution is to update the `array` implementation to include this on the array div (L173). This means the `array` field can be used within the label without causing issues.

```
                onClick={(e) => {
                  e.preventDefault();
                }}
```

Reply:
@chrisvxd tbh i didnt know the differnce between `FieldLable` and `FieldLabelInternal` but i just reverted it

Reply:
@chrisvxd tbh i didnt know the differnce between FieldLable and FieldLabelInternal but i just reverted it and only kept the `FieldLabel` changes

Reply:
@chrisvxd tbh i didnt know the differnce between `FieldLabel` and `FieldLabelInternal` but i just reverted it and only kept the `FieldLabel` changes